tethys_services/backends/hs_restclient_helper.py

```diff
@@ -0,0 +1,29 @@
+import hs_restclient as hs_r
+from django.conf import settings
+
+
+def get_oauth_hs(request):
+    hs = None
+    error_msg_head = "Failed to initialize hs object: "
+
+    if request.user.social_auth.count() == 0:
+        raise Exception(error_msg_head + "Not logged in through OAuth")
+    if request.user.social_auth.count() > 1:
```

Comment:
@swainn 
I was thinking about this. If user's tethys account gets associated with multiple social accounts, say hydroshare and hydroshare_beta..., which hydroshare instance should be used to retrieve data in following operations...